CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
 cmake_minimum_required(VERSION 3.9)
```

Comment:
bazel? :D

Reply:
Would like to learn it but not sure it it's worth it here :man_shrugging: